src/types/order.d.ts

```diff
@@ -451,15 +451,15 @@ export interface OrdersEndpoint
 
   /**
    * Confirm payment intent
-   * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/payments/paying-for-an-order/elastic-path-payments-stripe.html#confirmation---post-confirm-the-security-validation-succeeded
+   * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/payments/transactions.html#post-confirm-payment-intent
    * @param orderId - The ID of the order.
    * @param transactionId - The ID of the transaction you want to confirm.
-   * @param body - The empty data object.
+   * @param body - The body of the order.
    */
   Confirm(
     orderId: string,
     transactionId: string,
-    body: {}
+    body: ConfirmPaymentBody
```

Comment:
i wonder if its clearer to do a union type @butikden `ConfirmPaymentBody | {}` so i can see the body should be one or the other

Reply:
I know what you mean but it probably won't be cleaner as long as /confirm endpoint for PayPal expects only `gateway` field in body
https://documentation.elasticpath.com/commerce-cloud/docs/api/payments/paying-for-an-order/paypal-express-checkout-payments.html#step-2-post-complete-the-payment

Reply:
and the current implementation of the `ConfirmPaymentBody` interface expects `method`, `gateway`, `payment` to be present